esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -1491,40 +1636,79 @@ bool CWsESDLConfigEx::onListDESDLEspBindings(IEspContext &context, IEspListDESDL
     return true;
 }
 
-bool CWsESDLConfigEx::onListESDLBindings(IEspContext &context, IEspListESDLBindingsRequest&req, IEspListESDLBindingsResponse &resp)
+static int bindingCompareFunc(IInterface * const *_itm1, IInterface * const* _itm2)
 {
+    IConstESDLBinding* itm1 = dynamic_cast<IConstESDLBinding*>(*_itm1);
```

Comment:
Should really be static casts (more efficient and more accurate).  Any dynamic cast needs to check that the result is not null.